cfunits/test/test_Units.py

```diff
@@ -4,6 +4,7 @@
 import unittest
 
 import numpy
+from dask.base import tokenize
```

Comment:
Since we are using `dask` here within the test modules, but not in the core library, we should add `dask` to the `tests_require` dependencies in the `setup.py`:

https://github.com/NCAS-CMS/cfunits/blob/76c6d7a6f100da226bb8a44e5a675ca1c688ddbb/setup.py#L96-L100

So please add it to that before we merge.

Reply:
Sorry - missed these, somehow. I'll address these, as you suggest, tomorrow.